src/HonzaBotner.Discord.Services/ServiceCollectionExtensions.cs

```diff
@@ -13,5 +14,12 @@ public static class ServiceCollectionExtensions
 
             return serviceCollection;
         }
+
+        public static IServiceCollection AddCommandServices(this IServiceCollection serviceCollection)
+        {
+            serviceCollection.AddSingleton<MuteRoleHelper>();
```

Comment:
This way the MuteRoleHelper can fetch the MuteRole ID directly. I felt like this solution was better, but let me know what you think. I could also hide it behind an interface, but I don't think it's really necessary (thoughts @tenhobi?)